src/extensions/extensionInterface.ts

```diff
@@ -269,6 +295,45 @@ export const recordingSelectionReducer: Reducer<RecordingSelection, RecordingSel
             ampScaleFactor: (state.ampScaleFactor || 1) * action.multiplier
         }
     }
+    else if (action.type === 'SetCurrentTimepointVelocity') {
+        return {
+            ...state,
+            animation: {
+                ...(state.animation || {}),
+                lastUpdateTimestamp: Number(new Date()),
+                currentTimepointVelocity: action.velocity
+            }
+        }
+    }
+    else if (action.type === 'AnimateRecording') {
+        const lastUpdate = state?.animation?.lastUpdateTimestamp || Number(new Date())
```

Comment:
Here is where the animation happens. Basically, we advance the current timepoint by an amount dependent on the velocity and how much time has elapsed since the last update.